### PR TITLE
Organizations: show audit logs

### PR DESCRIPTION
**Pull request by stsewd**
Similar to the user security logs, but from all the organization and according to their plan.
I was able to re-use the listing from the user security log :D.

Didn't want to overload the UI with all the information from each record (but you can get everything with the download button), still you can see some information when hovering over some places.

Screenshot 2021-10-18 at 18-26-58 Security Log - Read the Docs for Business
Screenshot 2021-10-18 at 18-26-43 Security Log - Read the Docs for Business
Screenshot 2021-10-18 at 18-26-31 Security Log - Read the Docs for Business

### CODE REVIEW

**Comment by stsewd** [comment]
@ericholscher I have added some docs and filtering by dates (`date_before/date_after` query args).
Some questions:

- Should we document the available query args and mention they can be combined? currently, they are implicit
- Should the download button respect the current filters? Right now it always downloads all records, without using the filters.
  I saw that github does respects the filters when you click on "export" https://github.com/settings/security-log.

File: docs/security-log.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+Security logs allow you to see what has happened recently in your organization or account.
+We store the IP address and the browser's User-Agent_ on each event.
+
+.. _User-Agent: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent
+
+User security log
+-----------------
+
+We store user security logs from the last 90 days, and track the following events:
```
[the remark this answers is not in the document]
Comment (stsewd) [reply docs/security-log.rst:12]: User securtiy logs are 90 days for both, we could respect the max period set by the organizations they belong on .com, but I have some questions if that happens, should we show those type of logs to org owners as well? https://github.com/readthedocs/readthedocs.org/pull/8620#discussion_r738178034

File: docs/security-log.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+.. note::
+
+   This feature exists only on :doc:`/commercial/index`.
+
+We store logs according to your plan, which can go from 90 days to unlimited,
+check our `pricing page <https://readthedocs.com/pricing/>`__ for more details.
+We track the following events:
+
+- Authentication on documentation pages from your organization
+- User access to every documentation page from your organization
```
[the remark this answers is not in the document]
Comment (stsewd) [reply docs/security-log.rst:38]: oops, we are currently storing this for pro plans, I'll add a new plan feature for this then.

File: readthedocs/profiles/views.py [hunk 3]
```diff
@@ -189,9 +187,17 @@ def get(self, request, *args, **kwargs):
             return self._get_csv_data()
         return super().get(request, *args, **kwargs)
 
+    def _get_start_date(self):
```
[the remark this answers is not in the document]
Comment (stsewd) [reply readthedocs/profiles/views.py:190]: That's already handled by the filter, the filter acts on top of this query.

**Comment by stsewd** [comment]
Download button now uses the current filters. But had to use a link instead of a form, as the form removes all query args from the forms' `action` attribute. To make the link look like a button, used the `button` class, but looks like that class doesn't exist on .com, so we need to figure out that first before merging...

Screenshot 2021-11-01 at 14-15-15 Security Log - Read the Docs for Business

**Comment by stsewd** [comment]
Fixed :D
It overlaps a little, but that looks like that happens in other elements on .com.

Screenshot 2021-11-01 at 16-47-32 Organization Security Log - Read the Docs for Business

**Review by ericholscher — APPROVED** [review]
Changes look good 👍 

Let's get this shipped and deployed.

**Pull request merged**